R/evaluate_condition.R

```diff
@@ -1,12 +1,22 @@
-#' Evaluates a condition
+#' Evaluates a conditio
 #'
-#' @param condi a \code{grader} \code{\link{condition}} object
+#' Evaluates the \code{\link{condition}} object to return a \code{\link{graded}} value.
+#'
+#' @param condi a \code{\link{condition}} object
```

Comment:
should this param now be called `condition`?

Reply:
yes. changed `condi` to `condition`